tests/Api/Responses/shop/checkout/completion/order_with_free_non_shippable_items.json

```diff
@@ -18,7 +22,11 @@
         "@type": "Address",
         "firstName": "John",
         "lastName": "Doe",
+        "phoneNumber": null,
```

Comment:
maybe we should use `skip_null_values` here 🤔

Reply:
https://github.com/Sylius/Sylius/pull/16681#discussion_r1705716320